### ISSUE TITLE
Mismatch between latest epub and gitbook

### ISSUE THREAD

**Issue by telemachus** [quii/learn-go-with-tests#427]
I think that the latest epub version is missing the chapter on iteration between Hello, World and Arrays and Slices.

![Screen Shot 2021-03-02 at 5 37 41 PM](https://user-images.githubusercontent.com/62958/109730441-78a7db00-7b87-11eb-9423-cc6df30c42d9.jpg)

On the other hand, the gitbook is missing the chapter Why Unit Tests and How to Make Them Work for You.

**Comment by quii** [comment]
Yeah the way it is organised between gitbook and the epub is a bit of a mess in truth, I'm not sure how to do sections in pandoc so it's kinda just a long list and it's not consistent with what gitbook does. This does need addressing

You can find the why chapter here on gitbook: https://quii.gitbook.io/learn-go-with-tests/meta/why

In respect to the integers chapter, i just built is locally and opened it and i could open the integers chapter. I notice you're using apple's Books app. I use it too! But I find sometimes it behaves oddly when you re-add books and things, I suspect some kind of caching. Could you try deleting it from your library and downloading the latest version [here](https://github.com/quii/learn-go-with-tests/releases/tag/11.2.0) and see if it makes a difference?

**Comment by telemachus** [comment]
@quii Thanks for the clarification about ordering in the gitbook and ebook. I'll keep that in mind—and sorry that I didn't look harder for it.

As for the ebook, I am on iBooks, but to clarify I'm missing the chapter on iteration not on integers. In the gitbook, the chapters run Integers, Iteration, Arrays and Slices, but in the ebook, it's just Integers and then Arrays and Slices.

In any case, I tried deleting and installing the latest ebook, but I'm still not seeing the chapter on Iteration there. I'll see if I can trash the cache of ibooks later and let you know.

**Comment by quii** [comment]
> but to clarify I'm missing the chapter on iteration not on integers

Ha! Surprised no one spotted this before. See https://github.com/quii/learn-go-with-tests/commit/bfbab9c60a7e88b0cd84335b1eb7a26eb3fb41b4

I'll cut a new release now...

**Comment by quii** [comment]
@telemachus https://github.com/quii/learn-go-with-tests/releases/tag/11.2.1 This page will have your new shiny epub with hopefully all the chapters included once Travis does its thing
